babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+  ],
+};
```

Comment:
waarom is deze nodig?

Reply:
bij het draaien van testen klaagt hij soms nog over het gebruik van imports buiten een module. Dit is een oplossing maar waarschijnlijk niet de beste. Ik denk dat ik dit best maandag eens met tom bekijk want mogelijks zitten er al in bestaande projecten oplossingen hiervoor.